suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -668,8 +666,18 @@ export const startDescriptorPreloadedDiscoveryThunk = createThunk(
         }
 
         // Start discovery for every network account type.
-        networksFiltered.forEach(network => {
-            dispatch(discoverNetworkBatchThunk({ deviceState, network }));
+        networksWithUnfinishedDiscovery.forEach(network => {
+            Object.values(network.accountTypes)
+                .filter(
+                    ({ accountType }: NetworkAccount) =>
+                        network.networkType !== 'cardano' ||
+                        (availableCardanoDerivations as string[]).includes(
+                            accountType ?? NORMAL_ACCOUNT_TYPE,
+                        ),
+                )
+                .forEach(({ accountType }: NetworkAccount) => {
+                    dispatch(discoverNetworkBatchThunk({ deviceState, network, accountType }));
+                });
```

Comment:
Change of logic here: previously `network` was duplicated for each `accountType`, so `dispatch` was sent for each network.

Now it's sent TODO foreach `network` (the `normal` account), and foreach of its accounts.